adaptive_scheduler/kernel_mappings.py

```diff
@@ -101,9 +101,10 @@ def get_rise_set_timepoint_intervals(rise_set_target, visibility, max_airmass, m
     try:
         rs_up_intervals = visibility.get_target_intervals(target=rise_set_target, up=True,
                                                           airmass=max_airmass)
-    except MovingViolation:
-        log.warn("rise-set failed on target")
-        log.warn(rise_set_target)
+    except MovingViolation as mv:
+        log.warning(repr(mv))
```

Comment:
Out of curiosity, why do you separate these warnings?
They are much harder to find separate when searching logstash.
Also, is there some way to include the target name and the request number in this error?

I'd love to see a message that looked something like 
```
log.warning(f"rise-set failed for request {reqnum} target {target.name} ({rise_set_target}): {repr(mv)}")
```

Reply:
I'll merge them into one log line. But unfortunately where this error takes place is inside a worker thread processing visibility stuff and it doesn't have any data other than target and visibility and constraints. I added the request id but only in the case where eccentricity >= 1.